### PR TITLE
chore(docs): readme.md and contributing doc updates

### PR DESCRIPTION
**Pull request by tabgok**
This commit contains two small changes to documentation:

1. Adds a WARNING message for installing git pre-commit hooks and instructions on how to uninstall.
2. Updates the link to the integration checklist template in the contributing to integrations document

Testing Strategy: N/A
Risks: None

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) are followed.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Author is aware of the performance implications of this PR as reported in the benchmarks PR comment.

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer is aware of, and discussed the performance implications of this PR as reported in the benchmarks PR comment.

### CODE REVIEW

**Review by emmettbutler — APPROVED** [review]

**Comment by emmettbutler** [comment]
@tabgok adding the `changelog/no-changelog` label to this PR will make the "validate changelog" CI step pass.

**Review by mabdinur — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ See [the contributing docs](https://ddtrace.readthedocs.io/en/stable/contributin
 
 ### Pre-commit Hooks
 
+#### WARNING: autohook.sh installation may install a few pre-commit hooks which are broken.
+If you have installed autohook.sh and git commands are failing, you can uninstall the script by deleting the created symlink with the command:
+
+    $ rm .git/hooks/pre-commit
+
+#### Installation
```
Comment (majorgreys) [inline README.md:43]: I would suggest a simpler uninstallation note to the end of this section. 

```suggestion
**NOTE**: If pre-commit hooks fail to run in your development environment, you can uninstall them by deleting the symlink created by the installation script:

    $ rm .git/hooks/pre-commit

#### Installation
```

**Review by majorgreys — COMMENTED** [review]
Can we open a bug report for this?

**Review by Yun-Kim — DISMISSED** [review]

**Comment by Yun-Kim** [comment]
Hi @tabgok, I've squashed your commits and signed them so that we can merge. Thanks for your contribution!

**Review by Yun-Kim — APPROVED** [review]

**Pull request merged**